e2e-test/helper.py

```diff
@@ -0,0 +1,198 @@
+
+import json
+import uuid
+import random
+import string
+import wds_client
+import requests
+import time
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+workspace_manager_url = ""
+rawls_url = ""
+leo_url = ""
+
+def setup(bee_name):
+# define major service endpoints based on bee name
+    global workspace_manager_url
+    workspace_manager_url = f"https://workspace.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"workspace_manager_url: {workspace_manager_url}")
+    global rawls_url
+    rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"rawls_url: {rawls_url}")
+    global leo_url
+    leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"leo_url: {leo_url}")
+    return workspace_manager_url, rawls_url, leo_url
+
+# CREATE WORKSPACE ACTION
+def create_workspace(cbas, billing_project_name, header, workspace_name = ""):
+    # create a new workspace, need to have attributes or api call doesnt work
+    rawls_workspace_api = f"{rawls_url}/api/workspaces";
+    request_body= {
+      "namespace": billing_project_name, # Billing project name
+      "name": workspace_name if workspace_name else f"api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}",
+      "attributes": {}};
+
+    workpace_response = requests.post(url=rawls_workspace_api, json=request_body, headers=header)
+    logging.debug(f"url: {rawls_workspace_api}")
+    logging.debug(f"response: {workpace_response}")
+    logging.debug(f"request_body: {request_body}")
+    assert workpace_response.status_code == 201, f"Error creating workspace: ${workpace_response.text}"
+    
+    # example json that is returned by request:
+    # {
+    #   "attributes": {},
+    #   "authorizationDomain": [],
+    #   "bucketName": "",
+    #   "createdBy": "yulialovesterra@gmail.com",
+    #   "createdDate": "2023-08-03T20:10:59.116Z",
+    #   "googleProject": "",
+    #   "isLocked": False,
+    #   "lastModified": "2023-08-03T20:10:59.116Z",
+    #   "name": "api-workspace-1",
+    #   "namespace": "yuliadub-test2",
+    #   "workspaceId": "ac466322-2325-4f57-895d-fdd6c3f8c7ad",
+    #   "workspaceType": "mc",
+    #   "workspaceVersion": "v2"
+    # }
+    workspace_response_json = workpace_response.json()
+    logging.debug(f"json response: {workspace_response_json}")
+    data = json.loads(json.dumps(workspace_response_json))
+    
+    logging.debug(f"data['workspaceId']:  {data['workspaceId']}")
+    
+    # enable CBAS if specified
+    if cbas is True:
+        logging.info(f"Enabling CBAS for workspace {data['workspaceId']}")
+        start_cbas_api = f"{leo_url}/api/apps/v2/{data['workspaceId']}/terra-app-{str(uuid.uuid4())}";

```

Comment:
nit: rename to `start_cbas_url`

Will requests.post log out the start_cbas_api?  If not, can you log the URL out?